### PR TITLE
Sort parsers and more

### PR DESCRIPTION
**Pull request by lipis**
With this rate that the parsers are being added, I think it makes sense to have them in alphabetical order..

### CODE REVIEW

File: tests_integration/__tests__/__snapshots__/early-exit.js.snap [hunk 1]
```diff
@@ -265,21 +265,21 @@ exports[`show detailed usage with --help no-semi (write) 1`] = `Array []`;
 exports[`show detailed usage with --help parser (stderr) 1`] = `""`;
 
 exports[`show detailed usage with --help parser (stdout) 1`] = `
-"--parser <flow|babylon|typescript|css|less|scss|json|graphql|markdown>
+"--parser <babylon|css|flow|graphql|json|less|markdown|scss|typescript>
```
Comment (azz) [inline tests_integration/__tests__/__snapshots__/early-exit.js.snap]: There was a logical grouping here that has broken

Comment (lipis) [reply tests_integration/__tests__/__snapshots__/early-exit.js.snap]: I can see that there are more errors and there are some trailing spaces that were removed..  do we really need those?

File: scripts/build/build.js [hunk 2]
```diff
@@ -11,11 +11,11 @@ const rootDir = path.join(__dirname, "..", "..");
 const parsers = [
   "babylon",
   "flow",
-  "typescript",
   "graphql",
```
Comment (azz) [inline scripts/build/build.js]: Leave the one as is,  otherwise there will be conflicts in #3536

**Comment by lipis** [comment]
Maybe it doesn't make sense if there is a huge refactor is about to happen

**Pull request closed without merging**